addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -81,6 +81,9 @@ export class MassMailingHtmlField extends HtmlField {
         if (this.props.readonly || !this.isRendered) {
             return super.commitChanges();
         }
+        if (!this._isDirty()) {
+            return;
+        }
```

Comment:
You mispelled `commitChanges` in the commit message. Considering you change this line in the next commit, I suggest squashing the two and merge their respective commit messages to have the full picture in a single commit.